resources/public/api/conf/1.0/employment_expenses_retrieve.yaml

```diff
@@ -0,0 +1,108 @@
+summary: Retrieve Employment Expenses
+description: |
+  This endpoint enables you to retrieve existing employment expenses.
```

Comment:
In the spec the description is `This endpoint allows a developer to retrieve existing employment expenses. A National Insurance number and Tax year must be provided.`